### PR TITLE
Update VM version know issue

### PR DESCRIPTION
**Pull request by abrarshivani**

### CODE REVIEW

File: documentation/known-issues.md [hunk 1]
```diff
@@ -23,5 +23,5 @@ This section lists the major known issues with Kubernetes vSphere Cloud Provider
 * For Kubernetes 1.6 and 1.7 releases (except Release v1.7.3 and onwards, Release v1.6.8 and onwards) vCenter Port other than 443 is not supported.
 
 ## Validation with Virtual Hardware version (VM version) 13
-* Validation of VCP against Container Hosts using Virtual Hardware version 13 is ongoing. [Link](https://github.com/vmware/kubernetes/issues/450)
+* VMs created from **ISO** with Virtual Hardware version 13 is not supported in Kubernetes release 1.9.0.
```
Comment (shahzebnp) [inline documentation/known-issues.md]: Just 1.9.0 is inaccurate, isn't it?
It won't work with 1.9.1 and 1.9.2 also...

File: documentation/known-issues.md [hunk 2]
```diff
@@ -23,5 +23,5 @@ This section lists the major known issues with Kubernetes vSphere Cloud Provider
 * For Kubernetes 1.6 and 1.7 releases (except Release v1.7.3 and onwards, Release v1.6.8 and onwards) vCenter Port other than 443 is not supported.
 
 ## Validation with Virtual Hardware version (VM version) 13
-* Validation of VCP against Container Hosts using Virtual Hardware version 13 is ongoing. [Link](https://github.com/vmware/kubernetes/issues/450)
+* VMs created from **ISO** with Virtual Hardware version 13 is not supported in Kubernetes release 1.9 but supported in earlier Kubernetes releases.
```
Comment (tusharnt) [inline documentation/known-issues.md]: I would reword it as follows -  Support for Virtual Hardware version 13 is broken only for VMs deployed via ISO due to a [known issue](https://github.com/vmware/kubernetes/issues/450).

**Review by tusharnt — CHANGES_REQUESTED** [review]

File: documentation/known-issues.md (hunk 2 above)
Comment (abrarshivani) [reply documentation/known-issues.md]: This is only for k8s release 1.9 so I think this needs to be mentioned.

Comment (divyenpatel) [reply documentation/known-issues.md]: We are in the process of cherry picking PR to 1.9 release. Once the PR is merged to 1.9 branch, we will not have this issue anymore. 

I would recommend to mention releases - 1.9.0 to 1.9.3. 
From 1.9.4 onward, If PR is cherry picked to 1.9 branch, we will not hit this issue.

Comment (abrarshivani) [reply documentation/known-issues.md]: PR is cherry-picked to 1.9 release branch. Will update this once v1.9.4 is cut

**Review by BaluDontu — APPROVED** [review]
